packages/runtime-dom/src/directives/vShow.ts

```diff
@@ -31,6 +31,7 @@ export const vShow: ObjectDirective<VShowElement> = {
         })
       }
     } else {
+      if (!oldValue === !value) return
```

Comment:
you probably wann move the `value !== oldValue` inside the first `if` and nest the last `else` without the inner if you just added inside of `if (transition)`

```js
if (transition) {
if ( value !== oldValue) {
} else {
setDisplay(..)
}
}
   
```

Reply:
DONE! Thank you for your hint